week01/ProductofArrayExceptSelf.cpp

```diff
@@ -0,0 +1,19 @@
+// TC: O(N)
+// MC: O(N)
+
+class Solution {
+public:
+    vector<int> productExceptSelf(vector<int>& nums) {
+        int suffixMult[nums.size() + 5];
```

Comment:
https://stackoverflow.com/questions/24136120/what-is-wrong-with-using-arrays-dynamically-allocated-in-c